packages/manager/fragments/account-sidebar/src/main.ts

```diff
@@ -0,0 +1,18 @@
+import { createApp, ref } from 'vue';
+import registerFragment from '@ovh-ux/ufrontend/fragment';
+import { Environment } from '@ovh-ux/manager-config';
+import { setupI18n } from '@/i18n';
+import AccountSidebar from '@/components/AccountSidebar';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@ovh-ux/ui-kit/dist/css/oui.css';
+
+registerFragment('account-sidebar').then(({ parent, config }: { parent: string; config: any }) => {
```

Comment:
`registerFragment` signature changed in `develop`: `config` object is replaced by `environment`.